scripts/docker/nginx-oss/ubuntu/entrypoint.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Can you move this file to scripts/docker/nginx-oss/ ? So that its consistent with the nginx-plus directory structure

Reply:
Same with the nginx.conf & the nginx-agent.conf. So as we add more dockerfiles for different OS we can reuse these files.